packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js

```diff
@@ -10,6 +10,7 @@ const {
 const getRegisterCompleteController = (req, res) => {
 	try {
 		const { session } = req;
+		const { caseRef } = session;
```

Comment:
I would be inclined to retrieve the case reference from the params. If we had a case where another controller did not have the caseRef added to the locals we would need to retrieve it from the params so following that rule should help to keep the codebase more consistent.

```suggestion
		const { params } = req;
		const { case_ref } = params;
```